pytest_mergify/__init__.py

```diff
@@ -1,4 +1,5 @@
 import typing
+import requests  # type: ignore[import-untyped]
```

Comment:
Can we install `types-requests` instead ?

Reply:
Now that we removed python 3.8 yes we can :D